python/lsst/ts/standardscripts/auxtel/daytime_checkout/slew_and_take_image_checkout.py

```diff
@@ -0,0 +1,225 @@
+        # This script does not require any configuration
+        pass
+
+    def set_metadata(self, metadata):
+        """Set estimated duration of the script."""
+
+        metadata.duration = 10.0 * 60  # Approximate 5min to completion.
+
+    async def run(self):
+
+        await self.assert_feasibility()
+
+        # Ensure mirrors and vents are closed for safer operations
+        await self.atcs.close_m1_cover()
+        await self.atcs.close_m1_vent()
+
+        # Enable Dome following to allow dome to slew with telescope
+        await self.atcs.enable_dome_following()
+
+        # Bias Verification, start with checkpoint for observer
+        await self.checkpoint("Bias Frame Verification")
+
+        # Turn on ATAOS correction(s)
+        cmd = await self.atcs.rem.ataos.cmd_enableCorrection.set_start(
+            m1=True, hexapod=True, atspectrograph=True
+        )
+        cmd_time = get_topic_time_utc(cmd)
+        self.log.info(f"ATAOS corrections enabled -- {cmd.result} at {cmd_time} UT")
+
+        # Now slew to a new position and start tracking a siderial target.
+        start_az = 45
+        start_el = 45
+        start_rot = 0
+        coord = self.atcs.radec_from_azel(az=start_az, el=start_el)
+        await self.atcs.slew_icrs(
+            coord.ra,
+            coord.dec,
+            rot=start_rot,
+            stop_before_slew=False,
+            rot_type=RotType.PhysicalSky,
+        )
+
+        # Take an Engineering test frame and verify ingestion at OODS
+        await self.checkpoint("Slew and take image 1/2")
+
+        self.latiss.rem.atoods.evt_imageInOODS.flush()
+        await self.latiss.take_engtest(2, filter=0, grating=0)
+        try:
+            ingest_event = await self.latiss.rem.atoods.evt_imageInOODS.next(
+                flush=False, timeout=STD_TIMEOUT
+            )
+        except asyncio.TimeoutError:
+            raise RuntimeError(
+                "Timeout waiting for imageInOODS event for image 1/2. This "
+                "usually means there is a problem with the image ingestion."
+            )
+
+        assert ingest_event.statusCode == 0, "Image ingestion 1/2 was not successful!"
+
+        ingest_event_time = get_topic_time_utc(ingest_event)
+        inst_setup = await self.latiss.get_setup()
+        self.log.info(
+            f"Image 1/2 with id {ingest_event.obsid} was ingested at "
+            f"{ingest_event_time.utc} UT with {inst_setup[0]} filter "
+            f"and {inst_setup[1]} grating"
+        )
+
+        # Pause and let it track for a few minutes
+        await asyncio.sleep(3.0 * 60.0)
+
+        await self.checkpoint("Slew and take image 2/2")
+
+        # Slew to a second target and repeat
+        start_az = 0.0
+        start_el = 80
+        start_rot = 0
+        coord = self.atcs.radec_from_azel(az=start_az, el=start_el)
+        await self.atcs.slew_icrs(
+            coord.ra,
+            coord.dec,
+            rot=start_rot,
```

Comment:
Same as before, maybe add a `target_name`?

Reply:
Change adopted.

Reply:
There was an issue with the first commit to address this comment, see [3981e87] for the fix.